setup.py

```diff
@@ -20,7 +20,7 @@
         "pytest-cov",
         "gammapy~=0.18",
         "ogadf-schema~=0.2.3",
-        "uproot4",
+        "uproot~=4.0",
         "awkward1",
```

Comment:
Sorry missed that. `awkward1` should now also be `awkward~=1.0`

Reply:
done